src/Redux/actionCreators/ServiceActions.js

```diff
@@ -24,22 +25,24 @@ export const fetchService = pagination => async dispatch => {
     });
 };
 
-export const executeService = (url, data) => dispatch => {
-  return fetch(url, {
-    method: "POST",
-    mode: "CORS",
-    body: JSON.stringify(data),
-  })
-    .then(response => response.json())
-    .then(data =>
+export const invokeServiceMethod = data => dispatch => {
+  Auth.currentSession({ bypassCache: true }).then(currentSession => {
+    const apiName = APIEndpoints.GET_SERVICE_LIST.name;
+    const path = "/call-service";
+    let myInit = {
```

Comment:
Should be const.

Reply:
[done](https://github.com/singnet/refactored-dapp/pull/13/commits/ffe9670c5b57721c102e6da6aaba2dd047791b0d)